tests/e2e/data_persistence.go

```diff
@@ -282,6 +282,119 @@ var _ = ginkgo.Describe("Data Persistence", func() {
 		}
 	})
 
+	// Data Persistence in case of Dynamic Volume Provisioning with XFS filesystem.
+	// Steps
+	//
+	// 1. Create SC with fstype set to "xfs"
+	// 2. Create a PVC which uses SC created in step1.
+	// 3. Create pod and wait for pod to become ready.
+	// 4. Verify volume is attached and volume is mouned inside pod with xfs filesystem.
+	// 5. Create empty file on the volume to verify volume is writable.
+	// 6. Verify newly created file exists on the volume.
+	// 7. Delete pod.
+	// 8. Create a new pod using the previously created volume and wait for pod to
+	//    become ready.
+	// 9. Create another empty file on the volume .
+	// 10. Verify newly created file and previously created files exist on the volume.
+	// 11. Delete pod.
+	// 12. Wait for volume to be detached.
+	// 13. Delete PVC and SC
+	ginkgo.It("[csi-block-vanilla] [csi-block-vanilla-parallelized] "+
+		"Dynamic volume provisioning data persistence test with XFS filesystem", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		var sc *storagev1.StorageClass
+		var pvc *v1.PersistentVolumeClaim
+		var err error
+
+		ginkgo.By("CNS_TEST: Running for vanilla k8s setup")
```

Comment:
this can be put under a `framework.Logf()` call

Reply:
done